ingestion-beam/bin/launch-22698806-direct

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+
+set -exo pipefail
+
+PROJECT="moz-fx-data-backfill-14"
+JOB_NAME="direct-read-test2"
+
+mvn compile exec:java -Dexec.mainClass=com.mozilla.telemetry.SimpleSink -Dexec.args="\
+    --runner=Dataflow \
+    --jobName=$JOB_NAME \
+    --project=$PROJECT  \
+    --inputType=bigquery_table \
+    --input=moz-fx-data-shared-prod:payload_bytes_decoded.telemetry_telemetry__main_v4 \
+    --bqReadMethod=storageapi \
+    --bqRowRestriction=\"CAST(submission_timestamp AS DATE) = '2020-04-15'\" \
```

Comment:
The four parameters above define the behavior of the input part of this pipeline. This script configures a job for direct reads (storage API).

All parameters below this line are irrelevant; they're there to configure the output of the Sink pipeline, but we've removed output for the purposes of this example.